main.tf

```diff
@@ -134,7 +134,7 @@ resource "aws_launch_configuration" "this" {
 
 module "bitbucketdb" {
   source  = "rhythmictech/rds-postgres/aws"
-  version = "~> 3.0.1"
+  version = "4.1.0-rc3"
```

Comment:
This version has merge conflicts, so I'm gonna block until this gets a full release